docs/diagnostics/PX1065.md

```diff
@@ -14,22 +14,23 @@ Acumatica Framework requires that all DAC field properties have a corresponding
 The PX1065 diagnostic checks whether the DAC field property has a corresponding BQL field in the definition of the DAC or in its base DACs.
 
 For DAC extensions, the diagnostic checks whether the DAC field property has a corresponding BQL field in the following locations:
- - The DAC extension itself and its base DACs from which it is derived.
- - The base DAC of the DAC extension and the DACs from which the base DAC has been derived.
- - The lower level DAC extensions to which the DAC extension is chained.
+ - The DAC extension itself and the base DACs from which it is derived
+ - The base DAC of the DAC extension and the DACs from which the base DAC has been derived
+ - The lower-level DAC extensions to which the DAC extension is chained
 
  ### Terminology
 
+[!NOTE]
```

Comment:
@EAndrosova is this `[!Note]` a some special markdown element? It doesn't look like it has any special display in the file preview. So, it looks a bit weird.

Reply:
fixed it, now it looks like a proper note